portal/views/organization.py

```diff
@@ -30,6 +31,7 @@
 @org_api.route('/organization')
 @crossdomain()
 @oauth.require_oauth()
```

Comment:
Might be out of scope for this PR, but do we actually need to require auth for this view?

Reply:
agreed that it's out of scope.  doesn't seem like there's a legit reason to require it.